_posts/2020-03-30-team-working-agreements.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: epic
+title: "Team Working Agreements"
+date: "2020-03-30"
+author: [steve-hicks, adam]
+categories: [people, culture, process, team]
+css: dialogue
+---
+
+Good team culture strives for cohesion. Once organizations get large enough, a tension emerges between the culture
+of individual teams and the culture of the larger organization. How do you achieve team cohesion across small teams
+_and_ the larger organizations they comprise?
+
+The culture at Artsy is driven by every team member, not mandated or handed down from above. This adds another
+level of tension, between individuals and their smaller teams. Team working agreements embrace that tension to
+provide a framework for converting tension into healthy culture.
+
+<!-- more -->
+
+Artsy Engineering is part of a larger organization we call PDDE (Product, Design, Data, and Engineering). PDDE is
+divided into several product teams, and each team contains at least one representative from product, design, data,
+and engineering. Each PDDE team delivers product solutions targeted to a specific product need.
+
+Until recently, those teams could expect a significant change in team members every quarter, as we encouraged
```

Comment:
I feel like something here around how all the changes happened at the same time might be helpful? Up to you!

```suggestion
Until recently, those teams could expect a significant change in team members at the start of every quarter, as we encouraged
```